packages/suite-web-landing/components/Index/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
```

Comment:
Let's discuss if we need to have both `Index` and `Start` components in different files. They are almost the same.